notebooks/depth_time_series.ipynb

```diff
@@ -0,0 +1,2310 @@
+{
```

Comment:
<p>It would be good to detail out where you got the taxonid value from.  Did you go to WoRMS directly? Did you use some service to get the identifier?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/iobis/pyobis/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Thank you @MathewBiddle for the review! To be honest, after fetching records so many times this value just got saved in my mind. But I'll edit the notebook and better replace it with the scientific name.

Reply:
Update: I have updated the notebook and inserted scientificname instead of taxonid.